### PR TITLE
repokitteh: implement retest azp

### PR DESCRIPTION
**Pull request by lizan**
Signed-off-by: Lizan Zhou <lizan@tetrate.io>

### CODE REVIEW

**Comment by lizan** [comment]
@itayd see the 4 comments above, this is ready to go. PTAL

File: ci/repokitteh/modules/azure_pipelines.star [hunk 1]
```diff
@@ -0,0 +1,49 @@
+load("github.com/repokitteh/modules/lib/utils.star", "react")
+
+_azp_context_prefix = "ci/azp: "
+
+def _retry_azp(organization, project, build_id, token):
+    """Makes an Azupre Pipelines Build API request"""
```
Comment (dio) [inline ci/repokitteh/modules/azure_pipelines.star:6]: nit. s/Azupre/Azure

**Comment by lizan** [comment]
@mattklein123 can you force merge this? I don't know why circle is stuck here

**Comment by mattklein123** [comment]
Sweet!

**Pull request merged**